### ISSUE TITLE
Other cpu types

### ISSUE THREAD

**Issue by evenoa** [stevemk14ebr/PolyHook#14]
Are there any plans for other cpu types, specifically
 Intel 960
 MPC 8245 (PowerPC)
 P2041 (PowerPC)

Regards, Even

**Comment by stevemk14ebr** [comment]
I don't plan on adding those myself. My long term plans are to first port this to osx and *nix. Then after that add arm support. If anyone did the work and testing to add other processors i would eagerly merge it.

**Comment by evenoa** [comment]
I can volunteer to test, but I suspect I would not be able to do the actual work without (a lot of ?)
hand holding and guidance as I have absolutely no asm experience.

**Comment by stevemk14ebr** [comment]
Better to let it be then.  For the intel 960 are you speaking of the i7-960?

**Comment by evenoa** [comment]
Not sure, my specification only says Intel 960, but the hardware is not recent. Regardless the PowerPC is more important to me.

**Comment by stevemk14ebr** [comment]
It would be a bit before i got to implementing that. I eventually do want to support power pc and arm but not anytime soon im afraid.

**Comment by evenoa** [comment]
Right, I understand completely, and will be waiting :-). In the mean time I might have a look.
